frontend/src/libs/components/protected-route/protected-route.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { Outlet } from 'react-router-dom';
+
+import { AppRoute } from '~/libs/enums/app-route.enum';
```

Comment:
use reexport. fix for all
```suggestion
import { AppRoute } from '~/libs/enums/enums';
```